TeachEasy/src/teacheasy/data/ButtonData.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Sam Hall and Sam Raeburn
+ * 
+ * Copyright (c) 2015 Sofia Software Solutions. All Rights Reserved.
+ * 
+ */
+package teacheasy.data;
+/** Class to hold Button Data 
+ * @version 1.0 05 Feb 2015
+ * @author sh1157 and sr896
+ * */
+public class ButtonData extends PageObject {
+	
+	public static enum ButtonInteractionType {
+		NORMAL
+	}
+	
+	private float xEnd, yEnd;
+	private float outlineThickness;
+	private boolean outline;
+	private boolean shadow;
+	private boolean visible;
+	private String text;
+	private int function;
+	private ButtonInteractionType interaction;
+	
+	/** Contructor for ButtonData */
+	public ButtonData(PageObjectType nType, float nXStart, float nYStart) {
+		super(nType, nXStart, nYStart);

```

Comment:
The type parameter of the super constructor should be a value from the PageObjectType enum, in this case PageObjectType.BUTTON. It should not be a parameter of the constructor of this data class, as all instances of this button data class MUST have type BUTTON.